neo4j/_async/driver.py

```diff
@@ -644,7 +644,6 @@ def session(self, **config) -> AsyncSession:
             """
             session_config = SessionConfig(self._default_workspace_config,
                                            config)
-            SessionConfig.consume(config)  # Consume the config
```

Comment:
Why remove the `SessionConfig.consume` call?

Reply:
Because the config options already get consolidated into a config object in the line above. This line caused warnings of deprecated options to be emitted a second time.